Zend/tests/name_collision_02.phpt

```diff
@@ -8,4 +8,4 @@ interface A { }
 
 ?>
 --EXPECTF--
-Fatal error: Cannot declare interface A, because the name is already in use in %s on line %d
+Fatal error: Cannot redeclare class A (previously declared in %s:%d) in %s on line %d
```

Comment:
This message is a bit misleading, as the interface A is tried to be declared and not class. Semantically the name `A` is redeclared, but `interface A` only declared, and we don't know what the user wanted to achieve.

Maybe something like this?
```suggestion
Fatal error: Cannot redeclare interface A (name A is used by class in %s:%d) in %s on line %d
```
or
```suggestion
Fatal error: Cannot declare interface A (name A is used by class in %s:%d) in %s on line %d
```

Reply:
It's actually the class that is redeclared, not the interface. The interface is the redeclaring element, which yeah is confusing. Hopefully, in reality this rarely diverges.